extensions/net/lib.deno_net.d.ts

```diff
@@ -117,6 +117,8 @@ declare namespace Deno {
     hostname?: string;
     /** Server certificate file. */
     certFile?: string;
+    /** Bypass certificate validation. */
+    bypassCertificates?: string | string[] | boolean;
```

Comment:
How does this field work? How is it related to the `--allow-insecure-cerfitifaces` flag?

Reply:
`bypassCertificates` deserialization is described above using the deserialize_allow_insecure_certificates` function. The `--allow-insecure-cerfitifaces` defines global list, which is then merged with `bypassCertificates`. Of course, above I wrote a question if this behavior is needed or we should just skip this and define a list only in the `GothamState` from the CLI.